packages/mobile/src/fiatExchanges/SimplexScreen.tsx

```diff
@@ -0,0 +1,170 @@
+import Button, { BtnSizes } from '@celo/react-components/components/Button'
+import colors from '@celo/react-components/styles/colors'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useLayoutEffect, useState } from 'react'
+import { useAsync } from 'react-async-hook'
+import { ActivityIndicator, StyleSheet, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { e164NumberSelector } from 'src/account/selectors'
+import { showError } from 'src/alert/actions'
+import { ErrorMessages } from 'src/app/ErrorMessages'
+import { numberVerifiedSelector } from 'src/app/selectors'
+import BackButton from 'src/components/BackButton'
+import WebView from 'src/components/WebView'
+import { CurrencyCode } from 'src/config'
+import ReviewFees from 'src/fiatExchanges/ReviewFees'
+import { fetchSimplexPaymentRequest, generateSimplexCheckoutForm } from 'src/fiatExchanges/utils'
+import { CURRENCY_ENUM } from 'src/geth/consts'
+import i18n from 'src/i18n'
+import { getLocalCurrencyCode } from 'src/localCurrency/selectors'
+import { emptyHeader, HeaderTitleWithBalance } from 'src/navigator/Headers'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import { navigateToURI } from 'src/utils/linking'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+export const simplexOptions = () => ({
```

Comment:
small nit: I know we're doing this in many places but for standardization and simplicity we're moving to put the options on `SimplexScreen.navigationOptions` which will avoid adding another import and help keep everything the same.